### PR TITLE
Adds lunchboxes

### PR DESCRIPTION
**Pull request by Yoshax**
- Adds lunchboxes
- Can take filled lunchboxes in loadout, these have a food item, a snack, and a drink
- Cleans up pathing in file
- Adds in-hand sprites for colors that don't have one, except rainbow because I suck at spriting

### CODE REVIEW

File: code/game/objects/items/weapons/storage/toolbox.dm [hunk 1]
```diff
@@ -70,13 +70,123 @@
...
-		new /obj/item/weapon/wirecutters(src)
-		new /obj/item/device/multitool(src)
+/obj/item/weapon/storage/toolbox/syndicate/New()
+	..()
+	var/color = pick("red","yellow","green","blue","pink","orange","cyan","white")
+	new /obj/item/weapon/screwdriver(src)
+	new /obj/item/weapon/wrench(src)
+	new /obj/item/weapon/weldingtool(src)
+	new /obj/item/weapon/crowbar(src)
+	new /obj/item/stack/cable_coil(src,30,color)
```
Comment (PsiOmegaDelta) [inline code/game/objects/items/weapons/storage/toolbox.dm]: I know you've just de-intended the line but the whole pick-color business could be replaced with
``` /obj/item/stack/cable_coil/random(src,30) ```

File: code/game/objects/items/weapons/storage/toolbox.dm [hunk 2]
```diff
@@ -70,13 +69,122 @@
...
+						/obj/item/weapon/reagent_containers/food/snacks/mushroompizzaslice, /obj/item/weapon/reagent_containers/food/snacks/vegetablepizzaslice,
+						/obj/item/weapon/reagent_containers/food/snacks/tastybread, /obj/item/weapon/reagent_containers/food/snacks/liquidfood,
+						/obj/item/weapon/reagent_containers/food/snacks/jellysandwich/cherry, /obj/item/weapon/reagent_containers/food/snacks/tossedsalad)
+
+		var/snack = pick(/obj/item/weapon/reagent_containers/food/snacks/donut/jelly, /obj/item/weapon/reagent_containers/food/snacks/donut/cherryjelly,
+						/obj/item/weapon/reagent_containers/food/snacks/muffin, /obj/item/weapon/reagent_containers/food/snacks/popcorn,
+						/obj/item/weapon/reagent_containers/food/snacks/sosjerky, /obj/item/weapon/reagent_containers/food/snacks/no_raisin,
+						/obj/item/weapon/reagent_containers/food/snacks/spacetwinkie, /obj/item/weapon/reagent_containers/food/snacks/cheesiehonkers,
+						/obj/item/weapon/reagent_containers/food/snacks/poppypretzel, /obj/item/weapon/reagent_containers/food/snacks/carrotfries
```
Comment (PsiOmegaDelta) [inline code/game/objects/items/weapons/storage/toolbox.dm]: Missing a , at the end of line.

**Comment by Datraen** [comment]
Can we get pictures for these?

**Comment by Yoshax** [comment]
Sprites courtesy of Ragnie: http://ss13polaris.com/forums/viewtopic.php?f=18&p=6613#p6601

**Comment by Datraen** [comment]
:+1:

**Comment by EmperorJon** [comment]
They cost 1 point, yes? I think maybe it should be upped to 2.

**Comment by Yoshax** [comment]
I meant to do that originally, but forgot.

**Comment by EmperorJon** [comment]
You only added the cost to the rainbow one :O

**Comment by Yoshax** [comment]
Oh jeeze, I'm terrible today. I meant to define it as a parent type, so the rest would inherit the cost.

**Comment by EmperorJon** [comment]
IT'S LUNCHTIME.

**Pull request merged**